main.tf

```diff
@@ -96,6 +98,9 @@ module "ecs-service-cloudwatch-dashboard" {
  * Create Elasticache subnet group
  */
 resource "aws_elasticache_subnet_group" "memcache_subnet_group" {
+  count = 1
```

Comment:
This is different than the previous PR, right? Does it need to be reverted?
I see you did it in different places below as well.